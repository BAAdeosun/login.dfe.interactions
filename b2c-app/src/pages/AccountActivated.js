import React from 'react';
import components from '../components';
import { POLICIES } from '../constants/policies';
import { LINK_TYPES } from '../constants/linkTypes';

class AccountActivated extends React.Component {

    render() {

        const content =
            <div>
                <components.Paragraph>You can start using your account to access your information.</components.Paragraph>
                <components.Link type={LINK_TYPES.BUTTON} policy={POLICIES.SIGNIN_INVITATION}>Sign in to your account</components.Link>
            </div>

        const title = "We've activated your account";

        const pageConfig = {
            title: title,
            header: title,
            aboveFormContent: content
        };


        return (
            <div id="accountActivated">
                <components.PageContainer pageConfig={pageConfig} />
            </div>
        )
    }

}

export default AccountActivated;