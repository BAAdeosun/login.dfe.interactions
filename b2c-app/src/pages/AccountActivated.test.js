import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import AccountActivated from './AccountActivated';

it('renders without crashing', () => {
    shallow(<AccountActivated />);
});

it('renders correctly', () => {
    const tree = renderer
        .create(<AccountActivated />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});