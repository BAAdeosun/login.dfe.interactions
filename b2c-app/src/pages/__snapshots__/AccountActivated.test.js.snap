// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  id="accountActivated"
>
  <div>
    <div
      className="govuk-width-container"
    >
      <div
        className="govuk-breadcrumbs"
      >
        <ol
          className="govuk-breadcrumbs__list"
        >
          <li
            className="govuk-breadcrumbs__list-item"
          >
            <a
              className="govuk-breadcrumbs__link"
              href="https://nationalcareers.service.gov.uk/"
            >
              Home
            </a>
          </li>
        </ol>
      </div>
      <main
        className="govuk-main-wrapper"
      >
        <div
          className="govuk-grid-row"
        >
          <div
            className="govuk-grid-column-two-thirds"
          >
            <h1
              className="govuk-heading-xl govuk-!-margin-top-6 govuk-!-margin-bottom-9"
            >
              We've activated your account
            </h1>
            <div>
              <p
                className="govuk-body"
              >
                You can start using your account to access your information.
              </p>
              <a
                className="govuk-button govuk-button--start"
                data-module="govuk-button"
                draggable="false"
                href={null}
                id="signInLink"
                onClick={null}
                role="button"
              >
                Sign in to your account
              </a>
            </div>
          </div>
        </div>
      </main>
    </div>
  </div>
</div>
`;
